todo-frontend-angular/src/app/todolist-selector/todolist/todolist.component.html

```diff
@@ -0,0 +1,12 @@
+<section>
+  <div class="btns-container">
+    <app-button [buttonConfig]="textBtnConfig" (onTextBtnClick)="onClickEventReceived()">
+    </app-button>
+  </div>
+</section>
+
+<div class="todos">
+  <div *ngFor="let todo of todos">
```

Comment:
If you don't use the div for styling, you could just add the *ngFor to the <app-todo> element :)

Reply:
redundant div removed